migrations/V87__Add_crud_role.sql

```diff
@@ -0,0 +1,20 @@
+DO
+$do$
+BEGIN
+  CREATE ROLE crud;
+  EXCEPTION WHEN DUPLICATE_OBJECT THEN
```

Comment:
This logic and exception handling is only needed for DEV. The `flyway clean` command we run there doesn't delete users or roles that were created by migration. Since we run `flyway clean` and apply all the migrations in DEV on each deployment, without this exception handling, this migration would error out on subsequent DEV deployments because the role already exists. The alternative was to drop and recreate the role if it already exists, but this approach seemed more conservative and potentially less destructive.